### ISSUE TITLE
Distribution changes for 7.0

### ISSUE THREAD

**Issue by jsi** [enonic/xp-distro#57]
*Filename changes:*
- [x] Latest version must be called unstable (not latest).
- [x] Unzipped directory must have same name as base filename.
- [x] The general distro shall be called enonic-xp-universal
- [x] The os-specific builds: `enonic-xp-<os>-<type>.zip`, where type is either sdk or server (used to be jdk or jre).
- [x] The CLI must be updated to reflect these changes.

*Welcome Tour changes:*
- [x] The last step must install Content Studio, Data Toolbox and Live Trace

**Comment by jsi** [comment]
Eller skal vi kalle den generelle distroen som Docker bruker, for `enonic-xp-generic`?

**Comment by jsi** [comment]
@GlennRicaud  found a way to discover latest without using unstable, but only after switching the text in the build file.  So, current solution is this:
 - We are building `unstable` , but we do no longer us it.

Instead, we do this in the Dockerfile to discover latest:
 - RUN XP_DISTRO_VERSION=`curl 'http://repo.enonic.com/api/search/latestVersion?g=com.enonic.xp&a=distro&v=*-SNAPSHOT'`

**Comment by jsi** [comment]
Second step has also been moved into its own issue: #59
